webview-ui/src/index.tsx

```diff
@@ -4,11 +4,20 @@ import "./index.css"
 import App from "./App"
 import reportWebVitals from "./reportWebVitals"
 import "../../node_modules/@vscode/codicons/dist/codicon.css"
+import posthog from "posthog-js"
+import { PostHogProvider } from "posthog-js/react"
+
+posthog.init(process.env.POSTHOG_PROJECT_API_KEY || "", {
+	api_host: process.env.POSTHOG_INSTANCE_ADDRESS || "",
+	debug: true,
+})
 
 const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
 root.render(
 	<React.StrictMode>
-		<App />
+		<PostHogProvider client={posthog}>
```

Comment:
Should we look at conditionally adding this to DOM based on `telemetryEnabled`?  Is there any negative impact by including this even when the users has not opted in?

Reply:
Yeah good question! So the reason why I made it this way is so that we can still use feature flags and other things that posthog provides

Reply:
:+1: